requirements.txt

```diff
@@ -0,0 +1,4 @@
+django==3.2.7
+django-crispy-forms
```

Comment:
Save also version for packages. Without version it will install always the latest one, not the version on which you created feature